shell/platform/android/io/flutter/embedding/engine/plugins/broadcastreceiver/BroadcastReceiverAware.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.embedding.engine.plugins.broadcastreceiver;
+
+import android.support.annotation.NonNull;
+
+/**
+ * {@link FlutterPlugin} that wants to know when it is running within a {@link BroadcastReceiver}.
+ */
+public interface BroadcastReceiverAware {
+  /**
+   * This {@code BroadcastReceiverAware} {@link FlutterPlugin} is now associated with a
```

Comment:
nit: passive voice, also see [the summary fragment](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment). Applies  to the below method too. 

```suggestion
   * Callback triggered when a {@code BroadcastReceiverAware} {@link FlutterPlugin} is associated with a {@link BroadcastReceiver}.
```

Reply:
done